Automattic/jetpack
# Writing prompt block: add example and quote style variation

## Proposed changes:

Adds example attributes to show a block preview from the inserter

<img width="459" alt="image" src="https://user-images.githubusercontent.com/1699996/226390192-5a6915d8-9fe1-4d64-b7d7-69512ae305dd.png">

Adds a quote style variation

<img width="635" alt="image" src="https://user-images.githubusercontent.com/1699996/226072414-da1e24d5-f332-4a3c-a3dc-b1f459853455.png">

Ensures content with the block is left aligned

<img width="220" alt="image" src="https://user-images.githubusercontent.com/1699996/226388599-81acad38-f0d0-42fa-b8b1-8fb1ca661b04.png">

Example avatars were copied from https://www.uifaces.co/browse-avatars/?provider%5B%5D=7 using the "Pexels.com" filter.

### Other information:


## Jetpack product discussion

PT: pe7F0s-uo-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Ensure your test site is connected to WP.com
* Enable experimental blocks (`define( 'JETPACK_EXPERIMENTAL_BLOCKS', true );`)
* Sandbox public-api and apply these changes to your wpcom sandbox with `bin/jetpack-downloader test jetpack adds/blogging-prompt-block-styles` (if not, the block header and footer will be empty, but it works, otherwise).
* Open the block inserter in the post editor and hover over the Writing Prompt block. You should see a preview of the block.
* Insert the block and go to the styles panel and select the Quote variation. Try the Spacing/Dimension controls and make sure they work as expected.
* Save the post and view on the front end.
